domains/rayyanjkhan.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "rayyanjkhan",
+        "email": "r.khan13245@kgs.edu.pk"
+    },
+    "record": {
+        "RECORDTYPE": "CNAME"
```

Comment:
This is not a real record, please read our [documentation](https://is-a.dev/docs).